### PR TITLE
docs: Document sitemap extension

### PR DESCRIPTION
**Pull request by Fryuni**

### CODE REVIEW

File: docs/src/content/docs/sitemap-ext.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
+---
+title: Sitemap Extensions
+description: An unofficial extension of the official @astrojs/sitemap integration.
+packageName: '@inox-tools/sitemap-ext'
+---
+
+An unofficial extension of the official [`@astrojs/sitemap`](https://docs.astro.build/en/guides/integrations-guide/sitemap/) integration that allows:
+
+- Each route to self-declare whether they should be included in the sitemap.
+- Pre-rendered dynamic routes to include only some of it's rendered pages in the sitemap.
```
Comment (hippotastic) [inline docs/src/content/docs/sitemap-ext.md:10]:
```suggestion
- Pre-rendered dynamic routes to include only some of their rendered pages in the sitemap.
```

File: docs/src/content/docs/sitemap-ext.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      postId: post.data.id,
+    }))
+  );
+});
+---
+```
+
+#### `setSitemap`
+
+This function receives an array of parameter maps with a boolean to indicate whether each of those parameters describe a route that should be included or excluded from the sitemap.
```
Comment (hippotastic) [inline docs/src/content/docs/sitemap-ext.md:128]: You doN't seem to describe what happens if the boolean is omitted. It seems to be optional based on the type definition.

**Review by hippotastic — COMMENTED** [review]
Looks really good! I just left two small comments on the docs.

File: docs/src/content/docs/sitemap-ext.md (hunk 2 above)
Comment (Fryuni) [reply docs/src/content/docs/sitemap-ext.md:128]: Nice catch

**Pull request merged**